client/src/screens/groups/components/GroupDetailsList.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Text,
+  FlatList,
+  Alert,
+} from 'react-native';
+
+import { Center, Container } from '../../../components/Container';
+
+import EventListItem from '../../../components/Events/EventListItem';
+import ScheduleListItem from '../../../components/Schedules/ScheduleListItem';
+import UserListItem from '../../../components/Users/UserListItem';
+
+
+const GroupDetailsRenderItem = ({ item }) => {
```

Comment:
nit: regular functions (that aren't supposed to 'appear' to be components or classes) should probably start with a lower case character (`groupDetailsRenderItem`)

Reply:
I always wondered what the rule was on that. TIL